server/server_test.go

```diff
@@ -112,7 +112,8 @@ func TestInitEngine(t *testing.T) {
 		{"hdd=", proto.Attributes{}, true, false},
 	}
 	for _, spec := range testCases {
-		ctx := &Context{Stores: spec.key}
+		ctx := NewContext()
+		ctx.Stores, ctx.GossipBootstrap = spec.key, "self://"
```

Comment:
`NewContext()` sets some default values that wouldn't be present with `&Context{}`.